util/zipdoc.go

```diff
@@ -83,7 +132,7 @@ func CreateZipDocument(id, srcPath string) (string, error) {
 
 func CreateZipDirectory(id string) (string, error) {
 	tmp, err := ioutil.TempFile("", "rmapizip")
-	log.Trace.Println("creating temp zip file:", tmp.Name())
+	// log.Trace.Println("creating temp zip file:", tmp.Name())
```

Comment:
Same as above, but if you want to keep the trace logging is fine, unless it's broken you should be able to set the logging level through an env variable.

Reply:
the logging was not initialized in the unit test (which failed with nil ref), was not sure how to solve it

Reply:
we'll need to add a null logger for tests at some point to do this right.